cli/coverage_cmd.go

```diff
@@ -0,0 +1,179 @@
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"go/build"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/tools/go/loader"
+
+	"github.com/kr/fs"
+
+	"sourcegraph.com/sourcegraph/go-flags"
+)
+
+func init() {
+	cliInit = append(cliInit, func(cli *flags.Command) {
+		_, err := cli.AddCommand("coverage",
```

Comment:
A better name might be sanity or validate? Coverage is a little overloaded due to the meaning it has from test coverage.